marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.test.tsx

```diff
@@ -52,6 +57,7 @@ describe('Update data load settings component', () => {
     expect(getByText('Please add target collections')).toBeInTheDocument();
     expect(getByText('Default Collections:')).toBeInTheDocument();
     expect(getByText('Target Permissions:')).toBeInTheDocument();
+    expect(getByText('Batch Size:')).toBeInTheDocument();
```

Comment:
can we add an assertion for the default value showing up in the document. One each in load-card.test and mapping-card.test?

Reply:
Default value is fetched from backend. I have tests for it where artifacts are created with 'batchSize' 100. Do you want me to add dummy data , set 'batchSize' to 100 there and see if it is rendered ?